client/asset/eth/eth.go

```diff
@@ -90,6 +90,25 @@ var (
 	minGasTipCap = dexeth.GweiToWei(2)
 )
 
+// WalletConfig are wallet-level configuration settings.
+type WalletConfig struct {
+	GasFeeLimit uint64 `ini:"gasfeelimit"`
+}
+
+// parseWalletConfig parses the settings map into a *WalletConfig.
+func parseWalletConfig(settings map[string]string) (cfg *WalletConfig, err error) {
+	cfg = new(WalletConfig)
+	err = config.Unmapify(settings, &cfg)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing wallet config: %w", err)
+	}
+	return cfg, nil
+}
+
+// Check that Driver implements asset.Driver.
+var _ asset.Driver = (*Driver)(nil)
+var _ asset.Creator = (*Driver)(nil)
```

Comment:
Oh, sorry for another iteration, but the resolution was to remove these duplicate lines because they are also below.

Reply:
My bad, I fixed it now.